test/utils/ValidatorTester.js

```diff
@@ -27,12 +28,17 @@ const ValidatorTester = (requestEndpoint) => (location) => (field_name) => ({
             const params = {};
             const res = await requestEndpoint(params);
 
-            checkCommonErrorResponse(res);
-            expect(res.body.errors).toContainEqual({
-                "location": location,
-                "msg": ValidationReasons.REQUIRED,
-                "param": field_name,
-            });
+            try {
+                checkCommonErrorResponse(res);
+
+                expect(res.body.errors).toContainEqual({
+                    "location": location,
+                    "msg": ValidationReasons.REQUIRED,
+                    "param": field_name,
+                });
+            } catch (e) {
+                throw new Error(`Failed isRequired test (${requestEndpoint}, ${location}, ${field_name})\n\n${e}`);
+            }
```

Comment:
It is still not necessary, it's just a helper to print the context when failing. I added it to all of the helpers as the use case is the same for all

Reply:
We should generalize it and make it a wrapper then, IMO.

The pattern would be something like pass the necessary context and a function into a method that does a try catch with the print of the error + context.

Otherwise this adds a lot of noise to the tests, which I think shouldn't be made "prettier for debugging" despite how good that sounds :stuck_out_tongue:

Reply:
I've looked at the updated code, this looks good now! :+1: